sdks/python/setup.py

```diff
@@ -159,6 +159,7 @@ def get_version():
 
 REQUIRED_TEST_PACKAGES = [
     'freezegun>=0.3.12',
+    'joblib>=1.1.0',
```

Comment:
Should we have `joblib` be in the `REQUIRED_PACKAGES`? technically it's being used in the regular `sklearn_loader.py` file.

Reply:
It is in required packages, but not in test packages for whatever reason.

Reply:
OK.  I've made joblib optional.

Unless its necessary I'll keep it out of REQUIRED_PACKAGES for now.